tests/runPylint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 if [[ "${CHECK}" == "pylintPY3K"  ]]; then
-  find WebApp Lib Core scripts -name "*.py" -and -not -name 'pep8_*' -exec pylint -E --rcfile=.pylintrc3k --py3k --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" {} +
+  find src/WebAppDIRAC/* -name "*.py" -and -not -name 'pep8_*' -exec pylint -E --rcfile=tests/.pylintrc3k --py3k --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" {} +
```

Comment:
No need for the `*`:
```suggestion
  find src/WebAppDIRAC/ -name "*.py" -and -not -name 'pep8_*' -exec pylint -E --rcfile=tests/.pylintrc3k --py3k --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" {} +
```